test/integration/controller/issuer/issuer_suite_test.go

```diff
@@ -0,0 +1,164 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package issuer_test
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/gardener/cert-management/pkg/apis/cert/v1alpha1"
+	certclient "github.com/gardener/cert-management/pkg/cert/client"
+	ctrl "github.com/gardener/cert-management/pkg/controller"
+	_ "github.com/gardener/cert-management/pkg/controller/issuer"
+
+	"github.com/gardener/controller-manager-library/pkg/controllermanager"
+	"github.com/gardener/controller-manager-library/pkg/controllermanager/cluster"
+	"github.com/gardener/controller-manager-library/pkg/controllermanager/controller/mappings"
+	"github.com/gardener/controller-manager-library/pkg/resources"
+	"github.com/gardener/controller-manager-library/pkg/utils"
+	"github.com/gardener/gardener/pkg/logger"
+	"github.com/go-logr/logr"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	kubernetesscheme "k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+func TestIssuerController(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Test Integration Issuer Controller Suite")
+}
+
+const testID = "issuer-controller-test"
+
+var (
+	ctx context.Context
+	log logr.Logger
+
+	restConfig     *rest.Config
+	testEnv        *envtest.Environment
+	testClient     client.Client
+	kubeconfigFile string
+
+	scheme *runtime.Scheme
+)
+
+var _ = BeforeSuite(func() {
+	// a lot of CPU-intensive stuff is happening in this test, so to
+	// prevent flakes we have to increase the timeout here manually
+	SetDefaultEventuallyTimeout(5 * time.Second)
```

Comment:
I think this might be a copied leftover. I commented it out locally, ran the integration test a couple of times, and didn't observe any `Eventually` timeouts.
```suggestion
```